### ISSUE TITLE
Confused about training the surrogate model(Pong)

### ISSUE THREAD

**Issue by lingjunz** [psuwuxian/rl_attack#7]
I'm trying to reproduce the results on the Pong Game. There are some questions about the implementations. 

According to RUNNING.md, I should run this command `python play_pong_train.py --save_victim_traj=True` to collect the victim’s observations and its corresponding actions; Then, the trajectories are used to train a mimic model (surrogate model) of the victim agent. 

But I noticed that in the source code,  there is a pre-trained mimic_model used to collect trajectories. It makes me confused. Could you please illustrate it more?

in play_pong_player0.py line 187, _black_box = True.
in pposgd_wrap.py line 80-81, self.pretrained_mimic = True
in pposgd_wrap.py line 144-147, need to load a pre-trained mimic model?

Q2: 
in pposgd_wrap.py, line 278: `action_modeling_mse = tf.reduce_mean(tf.square(tf.multiply(action_opp_mal, action_opp_next_ph)))`, why tf.multiply is used to compute mse loss instead of minus?

**Comment by nuwuxian** [comment]
For the Q1, in our paper, we need to train a mimic model (surrogate model) at first.  After that, we could do a black-box attack. So we provided our well-trained mimic model here. 
As for Q2, this line of code is useless, the state modeling loss is in line 271. I will refactor the Pong game source code sooner and you can refer to the MuJoCo code to have a better understanding of our proposed method.

**Comment by lingjunz** [comment]
Thanks for your kind reply. For Q1, how can I train a surrogate model from scratch (without provided mimic model)? 
The instructions in RUNNING.md seems to be a little contradictive since it relies on a well-trained mimic model to train a surrogate model.
![image](https://user-images.githubusercontent.com/42799742/111272650-dc450480-866d-11eb-8f07-eac5fd8d8796.png)

In my opinion, we need another agent (e.g., 2017may1 from agent zoo) to interact with the victim agent and then collect the corresponding observations and actions for the victim agent. Then, run  `python mimic_action.py`  to train a surrogate model.

Did I get that right?
